### PR TITLE
Updates the designs for all fair exhibitors

### PR DESCRIPTION
**Pull request by orta**
Before:
screen shot 2019-02-22 at 9 20 45 am

After:
screen shot 2019-02-22 at 9 44 25 am

### CODE REVIEW

File: src/lib/Scenes/Fair/Components/FairBoothPreview/__tests__/index-tests.tsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { Theme } from "@artsy/palette"
 import { renderRelayTree } from "lib/tests/renderRelayTree"
 import React from "react"
 import { graphql } from "react-relay"
-import { FairFixture } from "../../../__fixtures__"
+import { fairFixture } from "../../../__fixtures__"
 import { FairBoothPreviewContainer } from "../index"
```
Comment (orta) [inline src/lib/Scenes/Fair/Components/FairBoothPreview/__tests__/index-tests.tsx]: This isn't a class, so I lowercased it

File: src/lib/Scenes/Fair/Screens/__tests__/FairExhibitors-tests.tsx [hunk 2]
```diff
@@ -29,9 +29,27 @@ it("renders properly", async () => {
...
+  // The fixtured data is too big for me to hand-edit all of the exhibitors
+  // data into the shape we ask for
+  const anyExhibitors = exhibitors.exhibitors as any[]
+
+  // Because this uses flatlist you can't ask for all of the
+  // exhibitors in the array
+  expect(htmlDom).toContain(anyExhibitors[0].name)
+  expect(htmlDom).toContain(anyExhibitors[1].name)
+  expect(htmlDom).toContain(anyExhibitors[2].name)
 })
```
Comment (orta) [inline src/lib/Scenes/Fair/Screens/__tests__/FairExhibitors-tests.tsx]: This last test wasn't too valuable, as it'd for people to read a massive machine-readable string, so now it checks that the text we expect is on the screen instead

File: src/lib/Scenes/Fair/__fixtures__/index.ts [hunk 3]
```diff
@@ -62,18 +62,24 @@ export const FairFixture = {
...
-        "Annely Juda Fine Art",
-        "Anton Kern Gallery",
+        { name: "Acquavella Galleries", id: "Acquavella Galleries", profile_id: "Acquavella Galleries" },
+        { name: "Alan Cristea Gallery", id: "Alan Cristea Gallery", profile_id: "Alan Cristea Gallery" },
+        { name: "Alfonso Artiaco", id: "Alfonso Artiaco", profile_id: "Alfonso Artiaco" },
+        { name: "Altman Siegel", id: "Altman Siegel", profile_id: "Altman Siegel" },
+        { name: "Anat Ebgi", id: "Anat Ebgi", profile_id: "Anat Ebgi" },
+        { name: "Annely Juda Fine Art", id: "Annely Juda Fine Art", profile_id: "Annely Juda Fine Art" },
+        { name: "Anton Kern Gallery", id: "Anton Kern Gallery", profile_id: "Anton Kern Gallery" },
       ],
```
Comment (orta) [inline src/lib/Scenes/Fair/__fixtures__/index.ts]: The new shape of the data B-Z are still just strings, if people need that they can add it

File: src/lib/Scenes/Fair/Screens/__tests__/FairExhibitors-tests.tsx (hunk 2 above)
Comment (l2succes) [reply src/lib/Scenes/Fair/Screens/__tests__/FairExhibitors-tests.tsx]: Yup, makes sense

**Comment by l2succes** [comment]
Yes, that was bugging me out, thanks @orta 👍

**Pull request merged**